.github/workflows/build-wheels.yml

```diff
@@ -0,0 +1,30 @@
+name: Build Python
+
+on: [push, pull_request]
```

Comment:
Do we need to run this on every push? Or just a PR?

Reply:
Revised to only run on PR to develop